pkg/skaffold/output/log/log.go

```diff
@@ -96,5 +98,41 @@ func SetupLogs(stdErr io.Writer, level string, timestamp bool, hook logrus.Hook)
 		FullTimestamp: timestamp,
 	})
 	logrus.AddHook(hook)
+	setupStdLog(logrus.StandardLogger(), stdlog.Default(), lvl)
+	setupGGCRLogging(logrus.StandardLogger(), lvl)
 	return nil
 }
+
+// setupStdLog writes Go's standard library `log` messages to logrus at Info level.
+// This function uses SetFlags() to standardize the output format.
+func setupStdLog(logger *logrus.Logger, stdLogger *stdlog.Logger, lvl logrus.Level) {
+	stdLogger.SetFlags(0)
+	if lvl >= logrus.InfoLevel {
+		stdLogger.SetOutput(logger.WriterLevel(logrus.InfoLevel))
+	} else {
+		// Discard logs if we're not interested.
+		stdLogger.SetOutput(io.Discard)
+	}
+}
+
+// setupGGCRLogging enables go-containerregistry logging, mapping its levels to logrus.
+// The mapping is:
+// - ggcr Warn -> logrus Error
+// - ggcr Progress -> logrus Info
+// - ggcr Debug -> logrus Trace
```

Comment:
Is there a reason we send `Warn` to `logrus.Error` rather than to `logrus.Warn`, and `Debug` to `logrus.Trace` rather than `logrus.Debug`?

Reply:
I'm happy to change if we prefer.

My thinking was as follows:
- `ggcr` defines `Warn` as "non-fatal errors": https://github.com/google/go-containerregistry/blob/main/pkg/logs/logs.go#L24
- `ggcr` `Debug` logging is _very_ verbose, it includes all HTTP requests and responses, including non-sensitive headers and full payload.

Reply:
`s/full payload/full payload except for image layers/`

Reply:
that makes sense to me! Just wanted some clarification as it wasn't immediately obvious to me